pkg/reconciler/controller/transformation.go

```diff
@@ -114,21 +114,12 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, trn *transformationv1alp
 func (r *Reconciler) reconcileKnService(ctx context.Context, trn *transformationv1alpha1.Transformation) (*servingv1.Service, error) {
 	logger := logging.FromContext(ctx)
 
-	dest := trn.Spec.Sink.DeepCopy()
-	if dest.Ref != nil {
-		// To call URIFromDestination(), dest.Ref must have a Namespace. If there is
-		// no Namespace defined in dest.Ref, we will use the Namespace of the source
-		// as the Namespace of dest.Ref.
-		if dest.Ref.Namespace == "" {
-			dest.Ref.Namespace = trn.GetNamespace()
-		}
-	}
-
 	var sink string
-	uri, err := r.sinkResolver.URIFromDestinationV1(ctx, *dest, trn)
-	if err != nil {
-		logger.Infof("Sink resolution error: %v", err)
-	} else {
+	if trn.Spec.Sink != (duckv1.Destination{}) {
```

Comment:
This comparison will not work because the struct's fields are pointers.

Reply:
I remove what I said, it's fine with `!=` if Ref and URI are just compared to `nil` 😅

Reply:
why? it is working both on the controller that I deployed manually and on the [playground](https://play.golang.org/p/wlycdAvOjz3)

Reply:
sorry, missed the second message